.circleci/config.yml

```diff
@@ -501,7 +491,7 @@ jobs:
             python3 build.py -a --nocache --parallel
       - tag-push-docker:
           project: orc8r
-          images: "nginx|controller"
+          images: "nginx|controller|base"
```

Comment:
all those extra lines you have here may be caused by a bad rebase. 

The PR should only included